internal/blockprocessor/committer.go

```diff
@@ -346,21 +347,33 @@ func constructDBEntriesForDBAdminTx(tx *types.DBAdministrationTx, version *types
 			}
 			value = v
 
+			// for each DB, if index is defined, we need to create an
+			// index DB to store index entries for that DB
+			indexDB := &worldstate.KVWithMetadata{
```

Comment:
What would be the value associated with the index db "\_index\_dbName" ?
Are all indices included in the same "\_index\_dbName" under different key schemas?

Reply:
For now, there is a single index per database. For each database, there is an index database. To allow multiple indexes per database, later, we will introduce either namespaces or hash(index) as postfix for the indexDB name. Having a database per index to store index entries is very flexible to perform fine-grained index management (lock, dropping the whole database when the index is deleted, dropping the whole database when the index changes completely, etc...)

Reply:
OK, an index database per index definition sounds good. I understand that our first milestone is for a single index definition per db.